<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="outputBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>491</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;D2Coding&quot;;</string>
   </property>
   <property name="text">
    <string>공학용계산기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="equalButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
